drivers/clk/msm/clock-cpu-8936.c

```diff
@@ -170,8 +310,8 @@ static void get_speed_bin(struct platform_device *pdev, int *bin,
 
 	*bin = (pte_efuse >> 2) & 0x7;
 
-	dev_info(&pdev->dev, "Speed bin: %d PVS Version: %d\n", *bin,
-								*version);
+out:
```

Comment:
drivers/clk/msm/clock-cpu-8936.c: In function 'get_speed_bin':
drivers/clk/msm/clock-cpu-8936.c:313:1: warning: label 'out' defined but not used [-Wunused-label]